authorization/invitation/repository/invitation.go

```diff
@@ -78,6 +81,8 @@ type InvitationRepository interface {
 
 	ListRoles(ctx context.Context, id uuid.UUID) ([]rolerepo.Role, error)
 	AddRole(ctx context.Context, invitationId uuid.UUID, roleId uuid.UUID) error
+
+	FindByToken(ctx context.Context, identityID uuid.UUID, acceptToken uuid.UUID) (*Invitation, error)
```

Comment:
Could please replace the term `token` by `code` ? Because `token` has a very different meaning in our service :)

Reply:
Done